packages/e2e-tests/register/seedPhrase.spec.js

```diff
@@ -87,7 +88,7 @@ describe("Account Registration Using Seed Phrase", () => {
             .grantPermissions(["clipboard-read", "clipboard-write"])
             .catch(test.skip);
         // skip test on mainnet
-        if (process.env.WALLET_NETWORK === "mainnet") {
+        if (getWalletNetwork() === "mainnet") {
```

Comment:
Not a blocker, but in a follow-up PR I would like to see valid values for `networkId` in a `constants.js` file (which we will later import from a central, shared package)

Reply:
Will do in linkdrop PR 👍